src/hevm/src/EVM/UnitTest.hs

```diff
@@ -670,9 +670,12 @@ formatTestLog events (Log _ args (topic:_)) =
     Just t -> case (Map.lookup (wordValue t) events) of
                    Nothing -> Nothing
                    Just (Event name _ _) -> case name of
-                     "logs" ->
+                     "log" ->
```

Comment:
Like this @MrChico rather than 6fa91f1bca57037b40848c9b104f4a6675d4c69c

Reply:
I would rather keep `logs` as `bytes`, since that is the type they are given in test.sol